### PR TITLE
chore(external docs): Document additional ACLs for aws_s3 sink

### PR DESCRIPTION
**Pull request by jszwedko**
Added in https://github.com/timberio/vector/pull/3439

Signed-off-by: Jesse Szwedko <jesse@szwedko.me>

### CODE REVIEW

File: docs/reference/components/sinks/aws_s3.cue [hunk 1]
```diff
@@ -90,12 +90,14 @@ components: sinks: aws_s3: components._aws & {
...
+					"private":                   "Owner gets FULL_CONTROL. No one else has access rights (default)."
+					"public-read":               "Owner gets FULL_CONTROL. The AllUsers group gets READ access."
+					"public-read-write":         "Owner gets FULL_CONTROL. The AllUsers group gets READ and WRITE access. Granting this on a bucket is generally not recommended."
+					"aws-exec-read":             "Owner gets FULL_CONTROL. Amazon EC2 gets READ access to GET an Amazon Machine Image (AMI) bundle from Amazon S3."
+					"authenticated-read":        "Owner gets FULL_CONTROL. The AuthenticatedUsers group gets READ access."
+					"bucket-owner-read":         "Object owner gets FULL_CONTROL. Bucket owner gets READ access."
+					"bucket-owner-full-control": "Both the object owner and the bucket owner get FULL_CONTROL over the object."
+					"log-delivery-write":        "The LogDelivery group gets WRITE and READ_ACP permissions on the bucket. For more information about logs, see [Amazon S3 Server Access Logging](https://docs.aws.amazon.com/AmazonS3/latest/dev/ServerLogs.html)."
```
Comment (lucperkins) [inline docs/reference/components/sinks/aws_s3.cue:100]: Let's put this URL in `urls.cue`

File: docs/reference/components/sinks/aws_s3.cue [hunk 2]
```diff
@@ -90,12 +90,14 @@ components: sinks: aws_s3: components._aws & {
 			type: string: {
 				default: null
 				enum: {
-					"private":            "Owner gets FULL_CONTROL. No one else has access rights (default)."
-					"public-read":        "Owner gets FULL_CONTROL. The AllUsers group gets READ access."
-					"public-read-write":  "Owner gets FULL_CONTROL. The AllUsers group gets READ and WRITE access. Granting this on a bucket is generally not recommended."
-					"aws-exec-read":      "Owner gets FULL_CONTROL. Amazon EC2 gets READ access to GET an Amazon Machine Image (AMI) bundle from Amazon S3."
-					"authenticated-read": "Owner gets FULL_CONTROL. The AuthenticatedUsers group gets READ access."
-					"log-delivery-write": "The LogDelivery group gets WRITE and READ_ACP permissions on the bucket. For more information about logs, see [Amazon S3 Server Access Logging](https://docs.aws.amazon.com/AmazonS3/latest/dev/ServerLogs.html)."
+					"private":                   "Owner gets FULL_CONTROL. No one else has access rights (default)."
```
Comment (lucperkins) [inline docs/reference/components/sinks/aws_s3.cue:93]: Let's put the permission names in backticks for good measure

**Review by lucperkins — APPROVED** [review]
Two small nits, otherwise 👍🏼

**Comment by jszwedko** [comment]
@lucperkins thanks for the quick review! I pushed https://github.com/timberio/vector/pull/6087/commits/bd8ec433ef7c12f1cb946046cc3aa74ecb4b2622

**Pull request merged**